DECISION-RECORD.md

```diff
@@ -7,4 +7,97 @@ Wel loggen we de error.
 
 **Soms weten we wel al in welk zaaksysteem een bepaald gegeven te vinden is. Bij de communicatie tussen front-end en backend volgen we zoveelmogelijk de standaard api's. Hoe communiceren we extra informatie over het bronsysteem zonder van de api's af te wijken?**
 We kiezen ervoor een header mee te sturen.
-Het alternatief zou zijn, om deze informatie helemaal te negeren en altijd elk gegeven in alle beschikbare zaaksystemen op te zoeken. De belangrijkste overweging om dit niet te doen is, naast het grote aantal redundante calls, de complexiteit die ontstaat in het correct afhandelen van fouten en niet gevonden gegevens.
\ No newline at end of file
+Het alternatief zou zijn, om deze informatie helemaal te negeren en altijd elk gegeven in alle beschikbare zaaksystemen op te zoeken. De belangrijkste overweging om dit niet te doen is, naast het grote aantal redundante calls, de complexiteit die ontstaat in het correct afhandelen van fouten en niet gevonden gegevens.
+
+## Zaken bij bedrijven nieuwe stijl
+- Voorstel is om de query mogelijkheden op het `/zaken` endpoint te gebruiken. Eerst zoeken op rollen is geen goede oplossing omdat de zgw standaard geen mogelijkheid biedt om zaken te filteren op de url/uuid van een rol.
+- Voorstel is om nog NIET verder gebruik te maken van de `expand` functionaliteit. Het is niet functioneel noodzakelijk, vereist extra werk omdat de esuite dit niet ondersteunt en het levert niet extra werk op als we dit eventueel later alsnog willen realiseren als de performance onvoldoende blijkt.
+- Hieronder staat per type entiteit uit KvK in pseudo-code eerst welke query parameters we kunnen gebruiken, en daarna welke handmatige filtering we vervolgens nog moeten doen.
+
+### Rechtspersoon
```

Comment:
Dit lijkt me zinvolle input voor de realisatie, maar ik weet niet of een decision record daar de plek voor is.

Reply:
Ik vind het prima om het ergens anders vast te leggen. Ik had een plek nodig om het te laten reviewen en met Maykin te delen. Heb je een alternatief voorstel?